src/code42cli/cmds/cases.py

```diff
@@ -0,0 +1,246 @@
+from collections import OrderedDict
+
+import click
+from py42.exceptions import Py42BadRequestError
+from py42.exceptions import Py42NotFoundError
+
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.output_formats import OutputFormatter
+
+
+_HEADER_KEYS_MAP = OrderedDict()
+_HEADER_KEYS_MAP["name"] = "Name"
+_HEADER_KEYS_MAP["assignee"] = "Assignee"
+_HEADER_KEYS_MAP["status"] = "Status"
+_HEADER_KEYS_MAP["createdAt"] = "Creation Time"
+_HEADER_KEYS_MAP["findings"] = "Notes"
```

Comment:
The `show` command required a case number.
For that reason, we should include the case number in the `list` output, so that I can find a case when calling list and then copy the case number for using `show`.

Reply:
added.